### DISCUSSION TITLE
Sites de freelancers

### DISCUSSION THREAD

**Discussion by israelmarmar** [frontendbr/forum#1050] [Discussão]
Alguém já trabalhou ou tem experiência em sites de freelancer? como upwork ou workana? Estou pensado em começar a utiliza-los.

**Comment by renanlara** [comment]
Eu já testei o Workana. Não consegui fechar nada pois a concorrência é grande e a galera que já está lá há algum tempo tem mais credibilidade.

**Comment by marcelogbrito** [comment]
Workana a dificuldade é a grande concorrência que acaba empurrando muita pechincha. Upwork nunca experimentei mas vou tentar. Ouvi dizer que paga melhor que os daqui. Mas a plataforma que eu acho mais legal pra freela, mas que você não mencionou é o App Getninjas. Por que? Porque a quantidade de propostas por anuncio divulgado é 3 então é uma concorrência mais saudavel. O App trabalha com geolocalização mostrando quão proximo geograficamente está o anunciante de você e não te limita a comunicação ao site da plataforma como é o caso do workana. voce pode tratar com o cliente da forma que preferir. por telefone, whatsapp, telegram, pessoalmente, etc... Só que a liberação do telefone de contato é cobrada pelo getnijas(assim que eles ganham $). Eles vendem 1000 creditos por R$160,00 e cada cliente desbloqueado custa uma quantidade de creditos.

**Comment by renanlara** [comment]
Verdade @marcelogbrito, GetNinjas é bem bacana mesmo! Eu testei ele por pouco tempo, recebi alguns contatos mas não consegui fechar nada.

**Comment by israelmarmar** [comment]
Eu iria começar com o upwork, apesar de estar em inglês, é um site internacional que tem projetos do mundo todo e talvez a oferta de trampos é alta.

**Comment by rubenmarcus** [comment]
Upwork,Freelancer etc tem muito Chines e indiano, entre outros puxando o valor da hora pra 2 dolares a hora.
90% dos projetos é valor baixo da hora. Recomendo tentar sites como Toptal, mas ai o nível de exigência e conhecimento técnico é muito maior. E não é facil entrar. (eu mesmo ainda não entrei, mas vou tentando até entrar kkkk)
